### PR TITLE
ref(ui): Convert select all notice styles to emotion

### PR DESCRIPTION
**Pull request by scttcper**
Changed the `<a>` tag to a link styled button 

mobile
image

desktop
image

### CODE REVIEW

**Review by billyvg — COMMENTED** [review]
Nice, how does this look at other breakpoints?

**Comment by scttcper** [comment]
@billyvg good call, button didn't wrap inline like the old option

File: src/sentry/static/sentry/app/views/issueList/actions.tsx [hunk 1]
```diff
@@ -810,6 +808,21 @@ const IconPad = styled('span')`
...
+  text-align: center;
+  margin: 0 -1px -1px;
+  padding: ${space(0.5)} ${space(1.5)};
+  z-index: 2;
+`;
+
+const SelectAllLink = styled('a')`
+  margin-left: ${space(1)};
+  font-size: ${p => p.theme.fontSizeLarge};
+  color: ${p => p.theme.blue400};
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/issueList/actions.tsx:823]: Do we need this? We should use default link color imo

I think it'd be preferable to use `<Button[priority="link"]>` if possible, but I know there can by styling challenges with that. (This should not block PR)

File: src/sentry/static/sentry/app/views/issueList/actions.tsx [hunk 2]
```diff
@@ -810,6 +808,21 @@ const IconPad = styled('span')`
   top: ${space(0.25)};
 `;
 
+const SelectAllNotice = styled('div')`
+  background-color: #fffdf7;
+  border: 1px solid #e7d796;
+  text-align: center;
+  margin: 0 -1px -1px;
+  padding: ${space(0.5)} ${space(1.5)};
+  z-index: 2;
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/issueList/actions.tsx:817]: Can we use a z-index from theme?

File: src/sentry/static/sentry/app/views/issueList/actions.tsx [hunk 3]
```diff
@@ -810,6 +808,21 @@ const IconPad = styled('span')`
   top: ${space(0.25)};
 `;
 
+const SelectAllNotice = styled('div')`
+  background-color: #fffdf7;
+  border: 1px solid #e7d796;
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/issueList/actions.tsx:813]: Are there any theme colors we can use?

**Review by billyvg — APPROVED** [review]
This is good to merge, you could just make any follow-up changes as necessary IRT theme things

File: src/sentry/static/sentry/app/views/issueList/actions.tsx (hunk 1 above)
Comment (scttcper) [reply src/sentry/static/sentry/app/views/issueList/actions.tsx]: worked with robin on the colors

**Pull request merged**